### PR TITLE
Add travis CI

### PR DESCRIPTION
**Pull request by chenrui333**
Followup PR of #60

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - beta
+  - nightly
+
+before_script:
+  - rustup component add rustfmt
+
+script:
+  - cargo build --locked
+  - cargo test --locked
+  - cargo fmt --all -- --check
```
Comment (kornelski) [inline .travis.yml]: I'm not planning to use rustfmt's formatting. It's a dumb tool that applied unconditionally sometimes makes formatting worse and less readable.

Comment (chenrui333) [reply .travis.yml]: hmm, did not know that before. 😄 

I will remove it.

Comment (chenrui333) [reply .travis.yml]: Just removed it.

Now it is just simple and easy build/test. :)

**Comment by kornelski** [comment]
Nice, thanks

**Comment by chenrui333** [comment]
Thanks for the merge!

**Pull request merged**